turbinia/workers/docker.py

```diff
@@ -75,14 +75,17 @@ def GetContainers(self, evidence):
     try:
       json_string = subprocess.check_output(docker_explorer_command).decode(
           'utf-8')
+
+      if "is not a Docker directory" in json_string:
```

Comment:
Do we know what the underlying issue is here?  Does the directory not exist, or does it just not contain what docker explorer is expecting?   Since it's a bit brittle to check for specific strings in the output of another tool, could we potentially do a check on the directory before we try to run docker-explorer on it?   Maybe @rgayon has some thoughts/ideas there?

Reply:
Checking the existence of the folder makes complete sense. Will rewrite.